es-app/src/guis/GuiGamelistOptions.cpp

```diff
@@ -89,12 +89,25 @@ GuiGamelistOptions::GuiGamelistOptions(Window* window, SystemData* system) : Gui
 			if(!hasFavorite)
 			{
 				LOG(LogDebug) << "Nothing to show in selected favorites mode, resetting";
-				favorite_only->setState(false);			
+				favorite_only->setState(false);
 				Settings::getInstance()->setBool("FavoritesOnly", favorite_only->getState());
 			}
 		}
 	});
-	
+
+	// Toggle: Show hidden
```

Comment:
This needs to be in a `if(Settings::getInstance()->getString("UIMode") == "Full")` block, as it does not make sense to show the toggle box in any of the other modes.